Extra parameters bag (#1266)
AzureAD/microsoft-authentication-library-common-for-android: changelog.txt

- [MINOR] Adds new API to support ADAL/MSAL migration off of AuthenticationSettings#setSecretKey to a 'managed key' generated by Common (#1244)
- [PATCH] Improved performance of getAccounts() call by reducing number of cache round-trips (#1271)
- [PATCH] Perf: Use an Iterator over SharedPreferences entries via getAllFilteredByKey() (#1262)

> - [MINOR] Adds plumbing for new API parameters in MSAL to trigger client options (#1266)
> - [MINOR] Fall back to browser view if we can't open custom tabs. (#1270)

Author: ```suggestion
- [MINOR] Fall back to browser view if we can't open custom tabs. (#1270)
- [MINOR] Adds plumbing for new API parameters to reach the token endpoint (#1266)
```
